test/controllers/shared.controllers/director.residential.address.controller.unit.ts

```diff
@@ -26,7 +26,7 @@ import {
   UPDATE_DIRECTOR_RESIDENTIAL_ADDRESS_LINK_PATH
 } from '../../../src/types/page.urls';
 import { isActiveFeature } from "../../../src/utils/feature.flag";
-import { Request } from "express";
+import e, { Request } from "express";
```

Comment:
looks like a typo?

Reply:
good spot.  Tests still ran

Reply:
yeah, I would have expected to have code smell at least for unused import :D